spec/features/member_reviews_vendor_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+feature 'member posts vendor review', %{
+  As a reviewer
+  I want to be able to leave a review for a specific truck
+  So that I can share my experience with other users
+} do
+
+  scenario 'visitor tries to post a vendor review' do
+    vendor = FactoryGirl.create(:vendor)
+    visit vendor_path(vendor)
+    expect(page).to have_no_content('You')
```

Comment:
This expect statement is kinda vague to me as a reader. Where is the "You" supposed to be coming from (or not coming from)? It would be great if you could use some sort of text that's more descriptive as to what it's talking about.